provision-contest/ansible/admin.yml

```diff
@@ -69,16 +69,19 @@
       set_fact:
         dj_git_repo_scripts: "{{ DJ_GIT_REPO_SCRIPTS_RESTRICTED if WF_RESTRICTED_NETWORK else DJ_GIT_REPO_SCRIPTS }}"
 
-    - name: Create working copy of the domjudge-scripts repo
-      become: true
-      become_user: domjudge
-      # We use a different directory here to have one single 'upstream' and not have issues with it
-      git: repo={{ dj_git_repo_scripts }} dest=/home/domjudge/domjudge-scripts-checkout version=main accept_hostkey=yes update=no
+    - name: Download offline repos
+      when: WF_RESTRICTED_NETWORK
+      block:
+        - name: Create working copy of the domjudge-scripts repo
+          become: true
+          become_user: domjudge
+          # We use a different directory here to have one single 'upstream' and not have issues with it
+          git: repo={{ dj_git_repo_scripts }} dest=/home/domjudge/domjudge-scripts-checkout version=main accept_hostkey=yes update=no
 
-    - name: Create working copy of the wf2021 repo
-      become: true
-      become_user: domjudge
-      git: repo=git@cds:wf2021 dest=/home/domjudge/wf2021 version=master accept_hostkey=yes update=no
+        - name: Create working copy of the wf2021 repo
+          become: true
+          become_user: domjudge
+          git: repo=git@cds:wf2021 dest=/home/domjudge/wf2021 version=master accept_hostkey=yes update=no
```

Comment:
Should we make a variable out of `wf2021`?

Reply:
I've pushed a commit for this.